tests/pytests/test_short_read.py

```diff
@@ -554,8 +554,17 @@ def runShortRead(env, data, total_len, expected_index):
         conn.close()
 
         # Make sure replica did not crash
-        res = env.cmd('PING')
+        max_up_attempt = 60
```

Comment:
Do you know how much attempts it usually takes with the new change?

Reply:
No. RDBs are small so it should be a small number